sass/sirius-header.scss

```diff
@@ -0,0 +1,14 @@
+.sirius-header {
+  .moj-header__navigation-link {
+    &[aria-current="page"] {
+      color: govuk-colour("blue") !important;
+      font-weight: bold;
+      text-decoration: none !important;
```

Comment:
Does text decoration change in the hover state to underline? This might prevent that depending on how those styles are set. Might be an intentional part of the design that it loses hover, so checking.

Reply:
So it doesn't with this addition. It's also not meant to according to Ryan, unfortunately I can't use any of the built in styling (where it'd remove underline for a clicked link) as we haven't really clicked supervision but this is how it's supposed to look. As this only applies when it is the current page it won't remove the underline when supervision is showing on another page.